README.md

```diff
@@ -122,6 +122,7 @@ To ease development, you can also use `npm run compile-continuously` to have you
 
 > In this case, you could probably also make use of [`live-server`](https://www.npmjs.com/package/live-server).
 
+Or you can even run `npm run dev` for automatic page reloading on change at the url: `http://localhost:8080/webpack-dev-server/cout-embauche-widget`
```

Comment:
The URL is logged when running the command. Avoid hardcoding it here, as if it changes in a later `webpack-dev-server` version (or we adjust it through options), the doc will be out of date.
Simply state that the user should navigate to the URL that is outputted by the command.